eos-updater-test-checkpoint/meson.build

```diff
@@ -0,0 +1,24 @@
+# Copyright 2024 Endless OS Foundation, LLC
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+eos_updater_test_checkpoint_sources = [
+  'main.c'
+]
+
+eos_updater_test_checkpoint_deps = [
+  dependency('ostree-1', version: '>= 2019.2'),
```

Comment:
Can you change this to `ostree_dep` per 0104703e68bcb2442b386aa43beb27b0ec1ee6c2?

Reply:
I can't, because that patch isn't on `eos5.1`. Oh, but it's the very next commit after the current `eos5.1` so I'll rebase this branch onto it.

Reply:
Ah ha. I forgot about the eos5.1 base.